src/cli/commands/test/iac-local-execution/index.ts

```diff
@@ -45,12 +50,60 @@ export async function test(
 
     const policy = await findAndLoadPolicy(pathToScan, 'iac', options);
 
-    const filesToParse = await loadFiles(pathToScan, options);
-    const { parsedFiles, failedFiles } = await parseFiles(
-      filesToParse,
-      options,
+    let parsedFiles: IacFileParsed[] = [];
+    let failedFiles: IacFileParseFailure[] = [];
+    const tfVarSupport = await isFeatureFlagSupportedForOrg(
+      'iacTerraformVarSupport',
+      iacOrgSettings.meta.org,
     );
 
+    const validFileTypes = tfVarSupport.ok
```

Comment:
suggestion: I think a strong boolean name describes instantly what it the variable is about. not sure if this is more readable though 🤷 

```suggestion
      const isTFVarSupportEnabled = (await isFeatureFlagSupportedForOrg(
      'iacTerraformVarSupport',
      iacOrgSettings.meta.org,
    )).ok;

    // if TF vars enabled, valid files are all except terraform files
    const validFileTypes = isTFVarSupportEnabled
```